src/home/WalletHome.tsx

```diff
@@ -96,6 +98,27 @@ function WalletHome({ route }: Props) {
     )
   }
 
+  // Scroll Aware Header
```

Comment:
It might be worth abstracting this as we are now using the same pattern in two places.